packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/Hosts/modals/VSphereNetworkModal.tsx

```diff
@@ -186,6 +186,17 @@ export const VSphereNetworkModal: React.FC<VSphereNetworkModalProps> = ({
     };
   });
 
+  const onChangUser: (value: string, event: React.FormEvent<HTMLInputElement>) => void = (
+    value,
+  ) => {
+    dispatch({ type: 'SET_USERNAME', payload: value });
+  };
+  const onChangePassword: (value: string, event: React.FormEvent<HTMLInputElement>) => void = (
```

Comment:
```suggestion

  const onChangePassword: (value: string, event: React.FormEvent<HTMLInputElement>) => void = (
```
adding empty line between the methods